app/src/main/java/org/apache/taverna/mobile/data/local/DBHelper.java

```diff
@@ -184,6 +186,24 @@ public boolean updateFavouriteWorkflow(String id) {
         return false;
     }
 
+    public Observable<List<Workflow>> getFavouriteWorkflow() {
+        return Observable.create(new Observable.OnSubscribe<List<Workflow>>() {
+            @Override
+            public void call(Subscriber<? super List<Workflow>> subscriber) {
+                if (subscriber.isUnsubscribed()) return;
```

Comment:
Remove this subscribe checking don't need because we have single Subscription in Favorite Presenter.